.github/workflows/documentation.yml

```
@@ -0,0 +1,38 @@
+name: "Documentation"
+
+on: # yamllint disable-line rule:truthy
+  pull_request:
+    branches:
+      - "main"
+    paths:
+      - .github/workflows/documentation.yml
+      - docs/**
+  push:
+    branches:
+      - "main"
+    paths:
+      - .github/workflows/documentation.yml
+      - docs/**
+
+jobs:
+  validate-with-guides:
+    name: "Validate documentation with phpDocumentor/guides"
+    runs-on: "ubuntu-22.04"
+
+    steps:
+      - name: "Checkout code"
+        uses: "actions/checkout@v3"
+
+      - name: "Install PHP"
+        uses: "shivammathur/setup-php@v2"
+        with:
+          coverage: "none"
+          php-version: "8.2"
+
+      - name: "Install dependencies with Composer"
+        uses: "ramsey/composer-install@v2"
+        with:
+          dependency-versions: "highest"
+
+      - name: "Run guides-cli"
+        run: "vendor/bin/guides -vvv --no-progress docs/en /tmp/test --fail-on-log"
```

The `en` directory exists only in Doctrine.

```suggestion
        run: "vendor/bin/guides -vvv --no-progress docs /tmp/test --fail-on-log"
```